Formula/protobuf.rb

```diff
@@ -2,8 +2,8 @@ class Protobuf < Formula
   desc "Protocol buffers (Google's data interchange format)"
   homepage "https://github.com/protocolbuffers/protobuf/"
   url "https://github.com/protocolbuffers/protobuf.git",
-      :tag      => "v3.10.1",
-      :revision => "d09d649aea36f02c03f8396ba39a8d4db8a607e4"
+      :tag      => "3.11.0",
```

Comment:
This tag does not exist. Did you mean `v3.11.0`?

Reply:
Yes. I did. Fixed.